packages/ember-runtime/lib/mixins/array.js

```diff
@@ -740,10 +741,13 @@ EachProxy.prototype = {
 };
 
 function addObserverForContentKey(content, keyName, proxy, idx, loc) {
+  let arrayString;
+  if (DEBUG) { arrayString = toString(content); }
```

Comment:
Please remove the `arrayString` variable, and update the assertion message to call `${toString(content)}` instead.